html5/js/Window.js

```diff
@@ -351,13 +351,13 @@ class XpraWindow {
         this.pointer_last_y = event_.offsetY;
       }
     });
-    canvas.addEventListener("pointermove", (event_) => {
+    canvas.addEventListener("mousemove", (event_) => {
       this.debug("mouse", "pointermove:", event_);
```

Comment:
Please update the debug logging to match the code. 
BTW, why is this change needed here?

Reply:
I am pretty sure this is necessary for cursor lock, as pointermove changes when the pointer is moved (does not happen when locked), and mousemove changes when the mouse is moved (correct). I will update the logging. Sorry.